lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtLeg.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.routing.pt;
+
+import org.eclipse.mosaic.lib.geo.GeoPoint;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PtLeg {
+
+    public record PtStop(GeoPoint location, Long departureTime, Long arrivalTime) {}
```

Comment:
This needs some documentation, what do `departureTime` and `arrivalTime` mean? Arrival at the PtStop and departure from the PtStop?